dfetch/project/vcs.py

```diff
@@ -109,6 +114,20 @@ def update(self, force: bool = False) -> None:
         logger.debug(f"Writing repo metadata to: {self.__metadata.path}")
         self.__metadata.dump()
 
+
+    def apply_patch(self) -> None:
+        """ Apply the specified patch to the destination """
+
+        patch_set = fromfile(self.__project.patch)
+
+        if patch_set:
+            if patch_set.apply(0, root=self.__project.destination, fuzz=True):
+                self._log_project(f"Applied path \"{self.__project.patch}\"")
+            else:
+                self._log_project(f"Applying path \"{self.__project.patch}\" failed")
```

Comment:
maybe we should propagate this up as a runtime-error, then the user sees red texts and gets a non-zero return code.

```python
raise RuntimeError("Applying patch failed")
```

Reply:
Done,

should the feature file check return code?

Reply:
No, not concern of this feature in my opinion. What do you think?

Reply:
Other feature files also don't check return code, so I think it is ok.